sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableAsyncClient.java

```diff
@@ -79,68 +85,65 @@
     }
 
     /**
-     * returns the table name associated with the client
+     * Gets the name of the table.
      *
-     * @return table name
+     * @return The name of the table.
      */
     public String getTableName() {
         return tableName;
     }
 
     /**
-     * returns the account for this table
+     * Gets the name of the account containing the table.
      *
-     * @return returns the account name
+     * @return The name of the account containing the table.
      */
     public String getAccountName() {
         return accountName;
     }
 
     /**
-     * returns Url of this table
+     * Gets the absolute URL for this table.
      *
-     * @return Url
+     * @return The absolute URL for this table.
      */
     public String getTableUrl() {
         return tableUrl;
     }
 
     /**
-     * returns the version
+     * Gets the REST API version used by this client.
      *
-     * @return the version
+     * @return The REST API version used by this client.
      */
     public TablesServiceVersion getApiVersion() {
         return TablesServiceVersion.valueOf(implementation.getVersion());
     }
 
     /**
-     * creates new table with the name of this client
+     * Creates the table within the Tables service.
      *
-     * @return void
+     * If a table with the same name already exists within the service, an exception will be thrown.
+     *
+     * @return An empty reactive result.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<Void> create() {
         return createWithResponse().flatMap(response -> Mono.justOrEmpty(response.getValue()));
     }
 
     /**
-     * creates a new table with the name of this client
+     * Creates the table within the Tables service.
+     *
+     * If a table with the same name already exists within the service, an exception will be thrown.
```

Comment:
Usually we list off the RuntimeExceptions with `@throws ResourceNotFoundException`. etc. or possible status codes; ConfigurationAsyncClient is an example.

Reply:
Yep, that makes sense. I wasn't sure whether to do this as part of this PR or another one, because figuring out what exceptions may be thrown will mean some significant time digging into the implementations. But I think I should have time tomorrow to add that stuff.

Reply:
Resolved